src/main/java/com/starrocks/connector/flink/manager/StarRocksStreamLoadVisitor.java

```diff
@@ -125,11 +125,8 @@ public String getErrorLog(String errorUrl) {
 
     private String getAvailableHost() {
         List<String> hostList = sinkOptions.getLoadUrlList();
-        if (pos >= hostList.size()) {
-            pos = 0;
-        }
-        for (; pos < hostList.size(); pos++) {
-            String host = new StringBuilder("http://").append(hostList.get(pos)).toString();
+        for (; pos < pos + hostList.size(); pos++) {
```

Comment:
This will get into an infinite loop when the tryConnection keeps failing. Cause the `pos < pos + hostList.size()` will always be true.